main.js

```diff
@@ -36,10 +37,25 @@ BlogPost.prototype.renderTitle = function(parent) {
 };
 
 BlogPost.prototype.renderAuthor = function(parent) {
+  const anchorAuthor = document.createElement("a");
+  anchorAuthor.href = "#";
   const author = document.createElement("div");
+  author.dataset.path = this.author;
```

Comment:
let's change the names of the data attributes for title & author so they're more descriptive (like `data-author` instead of `data-path`). also change the const names that are used for these values in the event handlers. i think this will help clarify the handler code, especially as we add more types of handlers for different interactions like tag filtering